### ISSUE TITLE
Early alerts of Repair Completion in Admiral Dashboard

### ISSUE THREAD

**Issue by FeeltheBerner** [KC3Kai/KC3Kai#1695]
- Chrome Version 55.0.2883.87 m (32 bit)
- Windows 10 Home Version 1607 (64 bit)
- KC3 Version 30.12.3
- Off the Webstore
- i. Repair Ship.
  ii. Check back when it says complete.
  iii. Actual game says there's still ~53 seconds left.
  iv. Timer always off by the aforementioned amount.

**Comment by DynamicSTOP** [comment]
@FeeltheBerner it's not an issue. it's a well known fact.
http://kancolle.wikia.com/wiki/Repair
> You can shave off the last minute of repair time by switching to the docking screen when a ship has less than one minute of repair time to go.

**Comment by LostLogia4** [comment]
The timer is reduced to 30 seconds now, so you should set the Timer Allowance to that.

**Comment by dragonjet** [comment]
@LostLogia4 oh is the default offset 30 seconds now for all players?

**Comment by sinsinpub** [comment]
i donot think so. it depends on each server, and may changes on each time of maintenance... (ps. my server is still 5x secs
